.pre-commit-config.yaml

```diff
@@ -209,3 +209,10 @@ repos:
         entry: sphinx-lint
         additional_dependencies: [sphinx-lint]
         types: [rst]
+  # FIXME: Uncomment this hook after, "pre-commit" supports mirroring Golang
```

Comment:
The pre-commit hook which I commented out here is hacky & doesn't work as I expect it to. I created the hook for my personal needs but I would prefer it to be officially supported.

I started a couple of discussions on this aspect (see [this comment](https://github.com/pre-commit/pre-commit-mirror-maker/issues/164#issuecomment-1403594913) & [this one](https://github.com/errata-ai/vale/discussions/556)). And if no one wants to create a better Vale pre-commit hook, then I'll rewrite the one I maintain.

Reply:
Can you move the style files to your precommit hook? Would be great not to have this committed to this repo but simply fun it via a hook.

Reply:
The path to styles can be configured using the `StylesPath` value in the `.vale.ini` file. And this folder is usually distributed in the VCS for portability. But I'll see if it can be packaged into the pre-commit hook instead.

I'm waiting for a response from Vale's author about officially supporting a pre-commit hook.